src/twisted/mail/test/test_pop3.py

```diff
@@ -609,30 +871,48 @@ def setUp(self):
         p.connectionLost(failure.Failure(Exception("Test harness disconnect")))
 
 
-    def contained(self, s, *caps):
-        for c in caps:
-            self.assertIn(s, c)
-
-
     def test_EXPIRE(self):
-        self.contained(b"EXPIRE 60", self.caps, self.pcaps, self.lpcaps)
+        """
+        I{EXPIRE} is in the server's advertised capabilities.
+        """
+        contained(self, b"EXPIRE 60", self.caps, self.pcaps, self.lpcaps)
 
 
     def test_LOGIN_DELAY(self):
-        self.contained(b"LOGIN-DELAY 120", self.caps, self.pcaps, self.lpcaps)
+        """
+        I{LOGIN-DELAY} is in the server's advertised capabilities.
+        """
+        contained(self, b"LOGIN-DELAY 120", self.caps, self.pcaps, self.lpcaps)
 
 
 
 class TestRealm:
+    """
+    An L{IRealm} which knows about a single test account's mailbox.
+    """
     def requestAvatar(self, avatarId, mind, *interfaces):
+        """
+        Retrieve a mailbox for I{testuser} or fail.
+
+        @param avatarId: See L{IRealm.requestAvatar}.
+        @param mind: See L{IRealm.requestAvatar}.
+        @param interfaces: See L{IRealm.requestAvatar}.
```

Comment:
A `@raises: L{AssertionError} when requesting an C{avatarId} other than I{testuser}` would be nice here.